### PR TITLE
SEO-175690 .NET MAUI Barcode Generator External

### PR DESCRIPTION
**Pull request by ghost**
Hi @Girija-Rajendran ,
|Title|Description|
|--|--|
|Task Category| .NET MAUI Barcode Generator|
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|External link PR link| External link -https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-barcode-generator/pull/2
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/175690
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Barcode-Generator---Money-Keywords.aspx

 

| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|.net maui barcode generator|0-10 |1FT|improved on page changes |added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links|
|net maui qr code generator|10-100 |14FT|To improve our FT experience | demo link added|
|.net maui barcode reader |10-100|3FT|To move our FT ranking to 1st position | improved on page changes|
|maui barcode reader|10-100 |13FT|To improve our FT experience| added CSS classes alignment format, images aligned in zigzag form, added documentation & Example links, unwanted space removed|
|barcode scanner in .net maui| 0-10 |14FT| To improve our FT experience |added corresponding documentation links, arranged images in zigzag form|

Worked on 20 keywords in the .NET MAUI Barcode Generator and am keeping track of all keyword details in the link below:-
https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Barcode-Generator---Money-Keywords.aspx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,60 @@
...
-
-Refer to the following documentation for the Syncfusion .NET MAUI Barcode Generator Control: 
-https://help.syncfusion.com/maui/barcode-generator/getting-started
-
-Check out this GitHub example for the Syncfusion .NET MAUI Barcode Generator Control: 
-https://github.com/SyncfusionExamples/.NET-MAUI-Barcode-Generator-Getting-Started
+The [.NET MAUI Barcode Generator](https://www.syncfusion.com/maui-controls/maui-barcodes?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples) is a data visualization control that is used to generate and display data in the machine-readable format using industry-standard one-dimensional and two-dimensional barcodes.A quick-start project helps you integrate a Barcode Generator control in the .NET MAUI application by adding the Syncfusion MAUI Barcode Generator package from NuGet. This project contains simple code customizations, such as how to generate different Barcode types like QR-Code and Data Matrix.
```
Comment (Sharly-Atieno) [inline README.md]: Give space her..
image

File: README.md [hunk 2]
```diff
@@ -1,14 +1,60 @@
...
+
+## About Syncfusion .NET MAUI Controls
+Syncfusion's [.NET MAUI Controls](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples) library is the only suite that you will ever need to build an application since it contains over 40 high-performance, lightweight, modular, and responsive UI controls in a single package. In addition to ListView, we provide popular .NET MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples), [Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples), [Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples), and [Excel](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples).
+
```
Comment (Sharly-Atieno) [inline README.md:49]: Recheck on this
image

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Added space.

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:49]: Updated the correct control name.

**Review by Sharly-Atieno — APPROVED** [review]

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**